packages/core/src/Site/index.js

```diff
@@ -1300,11 +1303,67 @@ class Site {
 
           return publish(basePath, options);
         })
-        .then(resolve)
+        .then(() => {
+          const git = simpleGit({ baseDir: process.cwd() });
+          const options = {};
```

Comment:
Is it possible to move the previous `options` up to avoid duplication?

Reply:
I think we can use a more explicit variable name (maybe `deployOptions`?) to make it clearer what kind of `options` these are as well!